src/attr/_config.py

```diff
@@ -9,6 +9,9 @@
 def set_run_validators(run):
     """
     Set whether or not validators are run.  By default, they are run.
+
+    .. deprecated:: 21.3.0 will not be moved to new ``attrs`` namespace.
+        Use :func:`attr.validators.set_disabled()` instead.
```

Comment:
```suggestion
    .. deprecated:: 21.3.0 It will not be removed, but it also will not be
        moved to new ``attrs`` namespace. Use `attr.validators.set_disabled()`
        instead.
```